aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -45,3 +45,13 @@ proposals:
             secrecy_threshold_in_percentage: 50
             reconstruct_threshold_in_percentage: 66
             fast_path_secrecy_threshold_in_percentage: 67
+  - name: step_5_dispatchable_token
```

Comment:
we don't add features in the default release yaml file anymore, they will be enabled separately. if you want you can create a dispatchable.yaml file in the same folder